src/cmd/raw.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/tkhq/go-sdk/pkg/api/client/private_keys"
+	"github.com/tkhq/go-sdk/pkg/api/models"
+)
+
+var (
+	rawSignPayloadEncoding string
+	rawSignHashFunction    string
+	rawSignPayload         string
+)
+
+func init() {
+	rawCmd.Flags().StringVar(&signingKeyID, "signing-key", "", "name or ID of the signing key")
+
+	rawSignCmd.Flags().StringVar(&rawSignPayloadEncoding, "payload-encoding",
```

Comment:
might make sense to include the default in the help text (e.g. `encoding of the payload, defaults to "TEXT_UTF8"`)

Reply:
Oh nice! That covers it 💯